core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -66,28 +75,28 @@ object SpendReportingService {
           currency.getCurrencyCode,
           Option(startTime),
           endTime = Option(startTime.plusDays(1).minusMillis(1)),
-          subAggregation = subKey.map(key => aggregate(rowsForStartTime, SpendReportingAggregationKeyWithSub(key)))
+          subAggregation = subKey.map(key => aggregate(rowsForStartTime, key, None))
         )
       }
       .toList
 
     def byWorkspace(rows: List[FieldValueList], subKey: SubKey): List[SpendReportingForDateRange] = rows
       .groupBy(row => row.get("googleProjectId").getStringValue)
-      .map { case (googleProjectId, projectRows) =>
-        val workspaceName = workspaceProjectsToNames.getOrElse(
-          googleProjectId,
+      .map { case (projectId, projectRows) =>
```

Comment:
Before refactors, I was finding it difficult to read because of sheer volume of text floating around.

I just needed to take a second pass at the refactor to clean it up.